geode-assembly/src/distributedTest/java/org/apache/geode/management/internal/rest/ManagementRequestLoggingDUnitTest.java

```diff
@@ -64,14 +61,8 @@ public void checkRequestsAreLogged() throws Exception {
     RegionConfig regionConfig = new RegionConfig();
```

Comment:
Can you add a start for the appender otherwise it ends up with some weird ERROR lines in the log - I thought I had done that when I originally wrote this, but I guess not.

Reply:
done